Client/Frontend/Browser/Playlist/PlaylistPopoverView.swift

```diff
@@ -130,16 +128,26 @@ struct PlaylistPopoverView: View {
             }
             .buttonStyle(BraveOutlineButtonStyle(size: .small))
             .foregroundColor(Color(UIColor.secondaryButtonTint))
-        }.padding(EdgeInsets(top: 22.0,
-                             leading: 30.0,
-                             bottom: 22.0,
-                             trailing: 30.0))
+        }
+        .padding(EdgeInsets(top: 22.0,
+                            leading: 30.0,
+                            bottom: 22.0,
+                            trailing: 30.0))
+        .background(Color(.braveBackground))
     }
 }
 
 struct PlaylistPopoverView_Previews: PreviewProvider {
     static var previews: some View {
-        PlaylistPopoverView(state: .addedToPlaylist)
+        Group {
+            ZStack {
+                Rectangle()
+                    .foregroundColor(.black)
+                    .edgesIgnoringSafeArea(.all)
+                PlaylistPopoverView(state: .addedToPlaylist)
+            }
+            .previewDevice("iPhone 12 Pro")
+        }
```

Comment:
Same as other, embed in `PopupView` instead

Reply:
Embded inside https://github.com/brave/brave-ios/pull/4508/commits/a51ac89b29bda9b22be0eed68d06ded22e4dfad0